package/MDAnalysis/core/topologyattrs.py

```diff
@@ -416,8 +457,16 @@ def get_segments(self, sg):
 
 #TODO: update docs to property doc
 class Resids(ResidueAttr):
+    """Residue ID"""
     attrname = 'resids'
     singular = 'resid'
+    transplants = defaultdict(list)
+
+    def atomget(atom):
+        return atom.residue.resid
+
+    transplants['atom'].append(
+        ('resid', property(atomget, None, None, atomget.__doc__)))
```

Comment:
Do we want this kind of transplant? Although it introduces a convenience, it does mean now we have more ways to maintain of getting the same information, and suggests that `Atom` objects should expose all `Residue` and `Segment` attributes, `Residue` objects should expose all `Segment` attributes, etc. I think it can be done, but is it a pattern we want to establish, and does this create complications for us later?